internal/product/accessor.go

```diff
@@ -301,6 +307,18 @@ func (p *postgresProductAccessor) GetAllProductVendors(
 	}, nil
 }
 
+func (p *postgresProductAccessor) getPriceByPVID(
```

Comment:
Please explain why some function names use `PascalCase` and some use `camelCase`?

Reply:
because in Go, having a capital case on the first character exposes it to other packages

since the `getPriceByPVID` is only used for the product package for now, I thought it's unnecessary to expose it

Reply:
ooh I just found out in Go the use of capital case on first character has meaning